reference/docs-conceptual/learn/deep-dives/everything-about-error-handling.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Everything you wanted to know about exceptions
+description: Error handling is just part of life when it comes to writing code.
```

Comment:
Why is this file in the PR

Reply:
Not supposed to be, must have been just an accidental transfer from another branch. I'll review more carefully